src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser.md

```diff
@@ -11,53 +11,81 @@ Note that because k6 does not run in NodeJS, the browser module APIs will slight
 
 </Blockquote>
 
-## Modules
+## Importable Properties
 
 The table below lists the importable properties from the top level module (`'k6/experimental/browser'`).
 
 | Property | Description                                                                                                                                                                          |
 |----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| chromium | A [BrowserType](/javascript-api/k6-experimental/browser/browsertype) to launch tests in a Chromium-based browser.                                                                                |
+| browser | The browser module API is the entry point for all your tests. See the [example](#example) and the [API](#browser-module-api) below. |
 | devices  | Returns predefined emulation settings for many end-user devices that can be used to simulate browser behavior on a mobile device. See the [devices example](#devices-example) below. |
-| version  | Returns the version number of k6 browser.                                                                                                                                           |
 
-### Devices Example
 
-To emulate the browser behaviour on a mobile device and approximately measure the browser performance, you can import `devices` from `k6/experimental/browser`.
+## Browser Module API
 
-<CodeGroup labels={["script.js"]} lineNumbers={[true]}>
+The browser module is the entry point for all your tests, and it is what interacts with the actual web browser via [Chrome DevTools Protocol](https://chromedevtools.github.io/devtools-protocol/) (CDP). It manages:
+- [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/) which is where you can set a variety of attributes to control the behavior of pages;
+- and [Page](/javascript-api/k6-experimental/browser/page/) which is where your rendered site is displayed.
+
+A new Chromium browser process is automatically launched when you use the `newContext` method of the `browser` module (`'k6/experimental/browser'`).
+
+| Method                                                                                    | Description                                                                                                                                           |
+|-------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [browser.close()](/javascript-api/k6-experimental/browser/close)                              | Closes the browser and all of its pages (if any were opened).                                                                                         |
```

Comment:
I believe we should not document `browser.close()` as it is no longer exposed, and is now handled by us, transparently to the user.

Reply:
See cdac40bb81b1f3142eb1b21dfddb29ec9b2a4dc9